Bottom: 5d14ce4aa534171ecda3f5056840930b2e128d26
Top:    a5a98e96f0793f089635e33128031e36d380c089
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-06-13 15:10:30 +0100

More auto-enforce DRC changes


---

diff --git a/src/line.c b/src/line.c
index 993a719..0658666 100644
--- a/src/line.c
+++ b/src/line.c
@@ -884,8 +884,12 @@ EnforceLineDRC (void)
     }
   END_LOOP;
 
-  rs.X = /*r45.X =*/ Crosshair.X;
-  rs.Y = /*r45.Y =*/ Crosshair.Y;
+  rs.X = Crosshair.X;
+  rs.Y = Crosshair.Y;
+#if 1
+  r45.X = Crosshair.X;
+  r45.Y = Crosshair.Y;
+#endif
 
   if (!TEST_FLAG (AUTODRCFLAG, PCB))
     {
@@ -905,7 +909,7 @@ EnforceLineDRC (void)
     }
   else
     {
-#if 1 /* Auto switch starting angle */
+#if 0 /* Auto switch starting angle */
       /* first try starting straight */
       r1 = drc_lines (&rs, false);
       /* then try starting at 45 */
@@ -927,7 +931,8 @@ EnforceLineDRC (void)
           Crosshair.Y = r45.Y;
         }
 #else /* Fixed starting angle */
-      drc_lines (&rs, (PCB->Clipping == 1) != gui->shift_is_pressed ());
+//      drc_lines (&rs, (PCB->Clipping == 1) != gui->shift_is_pressed ());
+      drc_lines (&rs, false);
       Crosshair.X = rs.X;
       Crosshair.Y = rs.Y;
 #endif
